codec/enforceproto/testdata/svc.pb.go

```diff
@@ -0,0 +1,441 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.23.0
+// 	protoc        v3.12.0
+// source: testdata/svc.proto
```

Comment:
These appears to be manually generated using golang proto. Please use gogo proto via `make proto-gen`.

Reply:
Indeed, I didn't know where or how to generate them for Cosmos but your direction helped, thank you!